### PR TITLE
[feat]:Quiz + Result Page

### PR DESCRIPTION
**Pull request by ultronjs**
-Added Quiz for 4 categories
-Added Result page with score and correct and incorrect answers

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,121 @@
-.App{
-  
+.quiz-score,
+.question {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-left: 10px;
+  color: #fff;
```
Comment (tusharkandpal) [inline src/App.css:7]: Please create color variables at root.

File: src/components/NavBar.js [hunk 2]
```diff
@@ -1,9 +1,30 @@
...
+      toast.error("Failed to Log Out", {
+        position: "bottom-right",
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      });
+      console.log(error)
```
Comment (tusharkandpal) [inline src/components/NavBar.js:25]: Remove console

File: src/components/ScoreCard.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import ScoreCardOption from "./ScoreCardOption";
+import {Alert} from "react-bootstrap"
+import "../App.css"
+
+function ScoreCard({questionsBank,saveSubmitAnswer}) {
+  console.log(questionsBank,saveSubmitAnswer)
```
Comment (tusharkandpal) [inline src/components/ScoreCard.js]: Here too.

File: src/pages/Login.js [hunk 4]
```diff
@@ -20,22 +24,35 @@ export default function Login() {
       setLoading(true);
       await login(emailRef.current.value, passwordRef.current.value);
       navigate("/");
-    } catch {
+    } catch(error) {
       console.log(error)
```
Comment (tusharkandpal) [inline src/pages/Login.js:28]: +1

File: src/pages/Login.js [hunk 5]
```diff
@@ -20,22 +24,35 @@ export default function Login() {
...
   }
 
+    async function handleGuestLogin(e) {
+      try {
+        setError("");
+        setLoading(true);
+        await loginWithGuestCreds(guestCreds.email,guestCreds.password)
+        navigate("/");
+      } catch(error) {
+        console.log(error);
```
Comment (tusharkandpal) [inline src/pages/Login.js:42]: +1

**Review by tusharkandpal — APPROVED** [review]
Code is clean but try to accommodate inline styling through classes.

**Review by Omkar-Chaskar — APPROVED** [review]
Nice UI and code is also clean and easy to understand.
Files are well structured.
Just small suggestion , try to avoid using inline css. 
Every thing is Great and Good to merge

File: src/components/NavBar.js (hunk 2 above)
Comment (ultronjs) [reply src/components/NavBar.js:25]: Done

File: src/components/ScoreCard.js (hunk 3 above)
Comment (ultronjs) [reply src/components/ScoreCard.js]: Done

File: src/pages/Login.js (hunk 5 above)
Comment (ultronjs) [reply src/pages/Login.js:42]: Done

File: src/App.css (hunk 1 above)
Comment (ultronjs) [reply src/App.css:7]: Done

**Comment by ultronjs** [comment]
Merging after 2 approvals

**Pull request merged**